scripts/helpers.js

```diff
@@ -109,12 +113,21 @@ hexo.extend.helper.register('github_file_url', function (path) {
   return urljoin(this.config.github.aframe.url, 'blob', aframeCurrentSha, path);
 });
 
+/**
+ * Get GitHub edit URL.
+ * - Change .html to .md.
+ * - For docs, must remove the active version (e.g., docs/0.2.0/guide.html -> docs/guide.md).
+ */
 hexo.extend.helper.register('website_github_edit_url', function (path) {
+  // For docs.
   if (path.indexOf('docs/') !== -1) {
-    return urljoin(this.config.github.aframe.url, 'edit', 'master', path.replace(/\.html$/, '.md'));
-  } else {
-    return urljoin(this.config.github.aframe_site.url, 'edit', 'master', this.config.source_dir, path.replace(/\.html$/, '.md'));
+    var activeVersion = path.split('/')[path.split('/').indexOf('docs') + 1];
```

Comment:
OK, cool. The string parsing is fragile, but the alternative is hard-coding a version to each doc page, or having the generator inject some page variables (in which we'd still have to do string parsing anyways). It's somewhat complex, I think this might be easiest.